CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+.. _v3-4-2:
+
+3.4.2 - 2021-02-08
+~~~~~~~~~~~~~~~~~~
+
+* Improvements to make the rust transition a bit easier. This includes some
+  better error messages and small dependency fixes. If you experience
+  installation problems **Be sure to update ``pip``** first, then check the
```

Comment:
This doesn't render the way you want https://github.com/pyca/cryptography/blob/72a6147d1f94ec9c66cd33e0757ed3b5f8b8261a/CHANGELOG.rst#342---2021-02-08